hack/build.sh

```diff
@@ -42,9 +42,16 @@ check_module_changes() {
 # Build terraform and providers only if needed
 build_terraform_and_providers() {
 	TARGET_OS_ARCH=$(go env GOOS)_$(go env GOARCH)
+	# shellcheck disable=SC2039
+	SKIP_TERRAFORM_PROVIDERS="${SKIP_TERRAFORM_PROVIDERS:"none"}"
 	bindir="${PWD}/terraform/bin/${TARGET_OS_ARCH}"
 	find "${PWD}/terraform/providers/" -maxdepth 1 -mindepth 1 -type d | while read -r dir; do
 		provider="$(basename "${dir}")"
+		# shellcheck disable=SC2039
+		if echo "${SKIP_TERRAFORM_PROVIDERS[@]}" | grep "$provider"; then
```

Comment:
I think disabling the shellcheck here is not correct. It's actually warning about a potential issue: sometimes `SKIP_TERRAFORM_PROVIDERS` is an array and others it's a string.

Reply:
Keeping it a string throughout.